application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/excel/SpreadSheetImportTest.java

```diff
@@ -282,11 +282,13 @@ private void assertFieldTypes() {
     }
 
     private void assertListFieldTypes(List<Map<String, Object>> fieldTypes) {
-
         assertThat(fieldTypes,
                    allOf(hasItem(allOf(hasColumn("jurisdiction_id", getIdForTestJurisdiction()),
                                        hasColumn("reference", "FixedList-marritalStatusEnum"),
                                        hasColumn("base_field_type_id", fieldTypesId.get("FixedList")))),
+                       hasItem(allOf(hasColumn("jurisdiction_id", getIdForTestJurisdiction()),
+                                       hasColumn("reference", "DynamicList-marritalStatusEnum"),
```

Comment:
dynamic list most of the times will be configured with empty FieldTypeParameter. I think we should have that kind of test here. 
Here we have tested FieldTypeParameter=marritalStatusEnum which is ok, we can leave it. But we must have also a test for FieldTypeParameter empty.

Reply:
Yes tested with both cases and added to IT excel sheet